tests/test-dirs/refactor-open/record_field.t

```diff
@@ -0,0 +1,25 @@
+FIXME refactor open rewriting the whole record instead of a field label
+  $ $MERLIN single refactor-open -action unqualify -position 4:7 ./ocaml-lsp-server/src/t.ml <<EOF
```

Comment:
Is the path here supposed to be `-filename <path>`?
If yes, let's just add the filename, and also, let's just make the path `test.ml`, because why not.